tools/validators/instance_validator/tests/telemetry_validator_test.py

```diff
@@ -32,26 +31,22 @@
 
 _MESSAGE_ATTRIBUTES_PATH_1 = path.join(_TELEMETRY_PATH,
                                        'message_attributes_CHWS_WDT-17.json')
-with open(_MESSAGE_ATTRIBUTES_PATH_1,
-          encoding='utf-8') as f:
+with open(_MESSAGE_ATTRIBUTES_PATH_1, encoding='utf-8') as f:
```

Comment:
pretty sure github linter will yell at you for this, I had to fix it the way it was